ImageGoNord/GoNord.py

```diff
@@ -439,20 +516,24 @@ def convert_image(self, image, save_path='', parallel_threading=False):
         original_image.close()
         pixels = self.load_pixel_image(image)
         is_rgba = (image.mode == 'RGBA')
-        if (parallel_threading == False):
-            self.converted_loop(is_rgba, pixels, original_pixels, image.size[0], image.size[1])
+
+        if (use_model):
+            image = self.convert_image_by_model(image, use_model_cpu)
         else:
-            step = ceil(image.size[0] / self.MAX_THREADS)
-            threads = []
-            for row in range(step, image.size[0] + step, step):
-                args = (is_rgba, pixels, original_pixels, row, image.size[1], row - step, 0)
-                t = threading.Thread(target=self.converted_loop, args=args)
-                t.daemon = True
-                t.start()
-                threads.append(t)
-
-            for t in threads:
-                t.join(timeout=30)
+            if (parallel_threading == False):
+                self.converted_loop(is_rgba, pixels, original_pixels, image.size[0], image.size[1])
+            else:
+                step = ceil(image.size[0] / self.MAX_THREADS)
+                threads = []
+                for row in range(step, image.size[0] + step, step):
+                    args = (is_rgba, pixels, original_pixels, row, image.size[1], row - step, 0)
+                    t = threading.Thread(target=self.converted_loop, args=args)
+                    t.daemon = True
+                    t.start()
+                    threads.append(t)
+
+                for t in threads:
+                    t.join(timeout=30)
 
         if (self.USE_GAUSSIAN_BLUR == True):
```

Comment:
I know that is not in the scope of this pr but:
```suggestion
        if self.USE_GAUSSIAN_BLUR:
```

Reply:
Nice catch